.github/actions/setup-sccache/action.yml

```diff
@@ -53,22 +53,23 @@ runs:
 
     - name: Authenticate to AWS
       id: aws-credentials
-      uses: aws-actions/configure-aws-credentials@v3
+      uses: aws-actions/configure-aws-credentials@v4.1.0
```

Comment:
Please use the revision hash to avoid supply chain issues.

Reply:
done.